src/main/java/frc/robot/commands/DefaultIntakeCommand.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.IntakeConstants;
+import frc.robot.subsystems.Input;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class DefaultIntakeCommand extends Command {
+
+  private boolean fullPower = false;
```

Comment:
Is fullPower a good name to use here? With what you implemented I agree, but what I was suggesting: a boolean to remove the pieceInIntake limitation, I don't think it applies as much. 

Plus, where's the pizzazz? The Dragon Ball reference? That last part is a joke but you get what I mean.